src/platform/with_wgl/native_gl_context.rs

```diff
@@ -54,10 +54,13 @@ impl Drop for NativeGLContext {
     fn drop(&mut self) {
         unsafe {
             if !self.weak {
+                self.unbind().unwrap(); //wglDeleteContext requires unbind
```

Comment:
Does this mean that the context to be deleted needs to be unbound? Or that there must be no current context.

I guess this is the first one, but please be clearer in this regard in the comment. Also, space after `//`.